crypto-conditions/README.md

```diff
@@ -109,11 +158,6 @@ Any contribution is very much appreciated!
 
 [![join the chat][rocketchat-image]][rocketchat-url]
```

Comment:
This Rocket Chat link syntax only seems to work in the branch root `README.md`. In this module's README, it seems broken.

Here are 2 alternate fixes suggestions:
* Visually identifical to the Rocket Chat link of the root documentation:
`[![Rocket.Chat](https://open.rocket.chat/images/join-chat.svg)](https://chat.hyperledger.org/channel/quilt)`
* Bringing precision (states the `hyperledger` chat server and its `quilt` channel):
`[![Rocket.Chat](https://open.rocket.chat/api/v1/shield.svg?type=channel&name=hyperledger&channel=quilt)](https://chat.hyperledger.org/channel/quilt)`